cmd/gctcli/futures.go

```diff
@@ -774,6 +816,15 @@ func getFundingRates(c *cli.Context) error {
 			return err
 		}
 	}
+	if c.IsSet("respecthistorylimits") {
+		respectFundingRateHistoryLimits = c.Bool("respecthistorylimits")
+	} else if c.Args().Get(7) != "" {
+		respectFundingRateHistoryLimits, err = strconv.ParseBool(c.Args().Get(6))
```

Comment:
```suggestion
		respectFundingRateHistoryLimits, err = strconv.ParseBool(c.Args().Get(7))
```

But will need to bumped by one anyway as per the above comment